### ISSUE TITLE
Deprecation warnings ruby 2.7.1 and rails 6.0.3

### ISSUE THREAD

**Issue by Aqualon** [petergoldstein/dalli#747]
Hi, after going from rails 6.0.2.2 to 6.0.3 I got the following new deprecation warnings

```
.rvm/gems/ruby-2.7.1/gems/dalli-2.7.10/lib/active_support/cache/dalli_store.rb:149: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
.rvm/gems/ruby-2.7.1/gems/activesupport-6.0.3/lib/active_support/cache/strategy/local_cache.rb:143: warning: The called method `write_entry' is defined here
.rvm/gems/ruby-2.7.1/gems/dalli-2.7.10/lib/active_support/cache/dalli_store.rb:420: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
.rvm/gems/ruby-2.7.1/gems/activesupport-6.0.3/lib/active_support/cache/strategy/local_cache.rb:122: warning: The called method `read_entry' is defined here
.rvm/gems/ruby-2.7.1/gems/activesupport-6.0.3/lib/active_support/cache/strategy/local_cache.rb:126: warning: Passing the keyword argument as the last hash parameter is deprecated
.rvm/gems/ruby-2.7.1/gems/dalli-2.7.10/lib/active_support/cache/dalli_store.rb:305: warning: The called method `read_entry' is defined here
.rvm/gems/ruby-2.7.1/gems/dalli-2.7.10/lib/active_support/cache/dalli_store.rb:167: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
.rvm/gems/ruby-2.7.1/gems/activesupport-6.0.3/lib/active_support/cache/strategy/local_cache.rb:153: warning: The called method `delete_entry' is defined here
.rvm/gems/ruby-2.7.1/gems/activesupport-6.0.3/lib/active_support/cache/strategy/local_cache.rb:155: warning: Passing the keyword argument as the last hash parameter is deprecated
.rvm/gems/ruby-2.7.1/gems/dalli-2.7.10/lib/active_support/cache/dalli_store.rb:332: warning: The called method `delete_entry' is defined here
```

**Comment by matthuhiggins** [comment]
Fixed if https://github.com/petergoldstein/dalli/pull/751 is merged.

**Comment by matthuhiggins** [comment]
Also fixed when ruby 2.6.2 is released: https://discuss.rubyonrails.org/t/new-2-7-3-0-keyword-argument-pain-point/74980

**Comment by kikonen** [comment]
Q: Is there something preventing merge of #751?
